tests/test_install_mender_sh.py

```diff
@@ -43,21 +47,46 @@ def script_server():
 
 
 @pytest.fixture(scope="function")
-def setup_tester_ssh_connection_f_apt_ready(setup_tester_ssh_connection_f):
-    # Explicitly accept suite change from "stable" to "oldstable"
-    setup_tester_ssh_connection_f.run(
-        "sudo apt-get update --allow-releaseinfo-change-suite"
-    )
+def generic_debian_container(request):
+    image = "debian:buster"
+    cmd = "docker run --network host --rm -tid %s" % image
+    output = subprocess.check_output(cmd, shell=True)
+
+    global docker_container_id
+    docker_container_id = output.decode("utf-8").split("\n")[0]
+
+    def finalizer():
+        cmd = "docker stop %s" % docker_container_id
```

Comment:
Should use `docker rm -f`, otherwise stopped containers will accumulate.

Reply:
Thanks.